cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -29,7 +29,45 @@ handle_cluster () {
   ./bin/hiveutil aws-tag-deprovision "${cluster_name}=owned" --region "${region}"
 }
 
-
+collect_metadata () {
+  local cluster_name
+  cluster_name=$1
+  ###eg cluster_name=kubernetes.io/cluster/ci-op-724qy8fn-55c01-s5c8w
+  local ns
+  ns=${cluster_name:22:14}
+  echo "cluster is used by namespace ${ns}"
+  echo "oc get pod -n ${ns}"
+  oc get pod -n ${ns}
+  for pod_name in $(oc get pod -n $ns -o=jsonpath='{.items[*].metadata.name}')
+  do
+    echo "inpecting ${pod_name} ..."
+    local env_cluster_name_value
+    ###eg env_cluster_name_value=ci-op-724qy8fn-55c01
+    env_cluster_name_value=$(oc get pod -n ${ns} ${pod_name} -o json | jq -r '.spec.containers[] | select (.name == "setup") | .env[] | select (.name == "CLUSTER_NAME") | .value')
+    if [[ -n "${env_cluster_name_value}" ]]; then
+      echo "find pod with setup container and env var CLUSTER_NAME ${env_cluster_name_value}"
+      if [[ ! ${cluster_name} == *"${env_cluster_name_value}"* ]]; then
+        echo "${env_cluster_name_value} is not contained in ${cluster_name}!"
+        continue
+      fi
+      if [[ ! ${env_cluster_name_value} == "${ns}-"* ]]; then
+        echo "${env_cluster_name_value} has not prefix ${cluster_name}-!"
+        continue
+      fi
+      echo "oc get pod -n ${ns} ${pod_name} -o yaml"
+      oc get pod -n ${ns} ${pod_name} -o yaml
+      echo "oc describe pod -n ${ns} ${pod_name}"
+      oc describe pod -n ${ns} ${pod_name}
+      for container_name in $(oc get pod -n ci-op-zf3n7jgn  e2e-aws-serial -o=jsonpath='{.spec.containers[*].name}')
+      do
+        echo "oc logs -n ${ns} ${pod_name} -c ${container_name}"
+        oc logs -n ${ns} ${pod_name} -c ${container_name}
```

Comment:
We have the logs in GCS already from the job -- we want to `describe` to get events for the Pod

Reply:
Will remove. describe is done above.

Reply:
Done.